electrum/payment_identifier.py

```diff
@@ -569,7 +595,7 @@ def _get_error_from_invoiceerror(self, e: 'InvoiceError') -> str:
                 error = _("Invoice requires unknown or incompatible Lightning feature") + f":\n{e!r}"
         return error
 
-    def get_fields_for_GUI(self) -> FieldsForGUI:
+    def get_fields_for_GUI(self, *, bip21_prefer_fallback=False) -> FieldsForGUI:
```

Comment:
Don't parameterize `get_fields_for_GUI`. It is fine to return a silent payment address here, even if the final payment is sent to a fallback address